From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@AndKjell__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (andkjell). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

PRmerger commented:
@AndKjell : Thanks for your contribution to the Azure documentation! The author, @andkjell, has been notified to review your proposed change.

billmath commented:
#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

KatieCumming commented:
@billmath - Hi Bill, please fix the merge conflict. See the [wiki](https://microsoft.sharepoint.com/teams/azurecontentguidance/wiki/Pages/Resolve%20a%20local%20merge%20conflict%20yourself.aspx) if you need instructions for fixing it. Also, please update yourself as the GitHub author in the metadata for signoff. Thanks, Katie
 #hold-off

AndKjell commented:
I've fixed the merge conflict, but I think the sign-off is supposed to happen in the -pr repo. Since I'm the author, I'll see if I can sign-off.

AndKjell commented:
#sign-off

ashishg2014 committed:
adding france geo

ashishg2014 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

mattbriggs committed:
API Profiles Ruby SDK

Commit:
updates to update tutorial

Commit:
updates to images and alerting

Commit:
updates

Commit:
updates to toc

Commit:
updates

Commit:
updates

Commit:
updates

Commit:
updates based on feedback

Commit:
correcting image

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into updatetutorial

lnalepa committed:
updated v2 registration portal doc

lnalepa committed:
Improve Acrolinx score

lnalepa committed:
Improve Acrolinx score

Commit:
Swap OS Disk

gled4er committed:
Adding paragrpah explaining approach to pass multiple parameters to an activity function.

gled4er committed:
Improving formatting.

gled4er committed:
Removing array sample and simplifying the Tuples one.

gled4er committed:
Improving wording.

rwike77 committed:
added connection example

ecfan committed:
Refactored functions and WDL content

ecfan committed:
Address Acrolinx issues

ecfan committed:
Add missing function + clarifications

ecfan committed:
Clarity edits

rwike77 committed:
adding articles for common name cert

rwike77 committed:
added deploy info

rwike77 committed:
updated toc

ecfan committed:
Clarity edits + update ToC

ecfan committed:
Clarity edits

ecfan committed:
Clarity edits

mattbriggs committed:
update with feedback from dev

ecfan committed:
ToC tweak

mattbriggs committed:
clearing merge

ecfan committed:
Metadata tweaks

gled4er committed:
Updating link to Tuples documentation.

ecfan committed:
Consistency edit

mattbriggs committed:
clearing a few stray typos

ecfan committed:
Formatting tweaks

rwike77 committed:
Merge branch 'cnfeature' of https://github.com/rwike77/azure-docs-pr into cnfeature

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cnfeature

rwike77 committed:
added text

rwike77 committed:
added template updates

rwike77 committed:
Acrolinx updates

rwike77 committed:
updates

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into updatetutorial

Commit:
updates to new readiness assement

Commit:
added additional language

Commit:
corrected borders on imageS

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cnfeature

rwike77 committed:
updating title

ecfan committed:
test formatting

gatneil committed:
added App Gateway section to VMSS Networing doc

ecfan committed:
Formatting tweak

Commit:
Adding CLI version

Commit:
weird file

ecfan committed:
test formatting

ecfan committed:
Revert formatting

ecfan committed:
Formatting edits

ecfan committed:
Formatting edits

ecfan committed:
Editing tweaks

ecfan committed:
Editing tweaks, add image

ecfan committed:
Fix formatting

ecfan committed:
Tweak formatting

ecfan committed:
Fix formatting

ecfan committed:
Formatting edits

ecfan committed:
Fix formatting

ecfan committed:
Edits

ecfan committed:
Editorial tweaks

ecfan committed:
Edits

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into updatetutorial

Commit:
updates based on feedback

ecfan committed:
Edits

ecfan committed:
Fix examples, add missing function

ecfan committed:
Fix function ordering

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into update-wdl-functions

ecfan committed:
Add back missing function

rwike77 committed:
Merge branch 'cnfeature' of https://github.com/rwike77/azure-docs-pr into cnfeature

ecfan committed:
Tweaks

ecfan committed:
Test subheadings

Commit:
updates based on feedback

gled4er committed:
Referencing Student Info in the sample code.

gled4er committed:
Improving sample code and adding links to C# 7 changes for Tuples.

rwike77 committed:
removed self-signed cert, fixed VM extension JSON

ecfan committed:
Add formatting suggestions

ecfan committed:
Reformat examples

ecfan committed:
Fix Acrolinx issues

ecfan committed:
Formatting edits

ecfan committed:
Tweak dates

ecfan committed:
Minor formatting tweaks

Commit:
size and storage note

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cnfeature

rwike77 committed:
minor fixes

rwike77 committed:
added cn support to create cluster via arm article

Commit:
tested cross rg swap

Commit:
acrolinx

Duncanma committed:
Fixing a typo in the TOC

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into cnfeature

rwike77 committed:
updated intro text, added note on self-signed certs

rwike77 committed:
new article for creating cert cn supprted cluster

rwike77 committed:
updated powershell

MashaMSFT committed:
added some clarity around where to find sql servers

MashaMSFT committed:
updated pic

v-stadam committed:
replace Markdown with YAML

v-stadam committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into TextAnalyticsLanding

v-stadam committed:
Replacing with YAML equivalent

v-stadam committed:
correct YAML syntax

v-stadam committed:
correct YAML syntax

v-stadam committed:
update TOC for new file

ashishg2014 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

v-viinde committed:
USER STORY 1245498 SaaS App Tutorial: Netsuite update

v-viinde committed:
updated

v-viinde committed:
Updated

ashishg2014 committed:
adding france ips

Commit:
correcting incorrect file in PR

Commit:
updates

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into updatetutorial

v-stadam committed:
metadata update

ash2017 committed:
Update iot-hub-compare-event-hubs.md

v-stadam committed:
metadata revert

billmath committed:
staging compat doc

ash2017 committed:
Update iot-hub-compare-event-hubs.md

Duncanma committed:
Merge branch 'master' into TOCTypoFix

v-stadam committed:
changes per review

v-stadam committed:
fix broken links

Commit:
tableau01

v-stadam committed:
fix broken links 2

JasonWHowell committed:
removing old information and shuffling content in these 3 related articles

v-deken committed:
Updated Bing Videos Search landing page to match Bing News Search

JasonWHowell committed:
Acrolinx

Commit:
tableau01

ecfan committed:
Address feedback

JasonWHowell committed:
Title adjustment

JasonWHowell committed:
Title adjustment

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into update-wdl-functions

JasonWHowell committed:
Title adjustment

ecfan committed:
Minor edit

v-deken committed:
Updated Bing Videos Search landing page to match Bing News Search

Commit:
tableau01

v-stadam committed:
change Node to Node.js

CelesteDG committed:
Updated the TOC to include the new file

Updated the TOC.md file to include the new file, active-directory-v2-registration-portal.md (App registration reference), that was added.

mattbriggs committed:
Merge branch 'master' into mdb_20180412_SDK_Ruby

Commit:
toc changes

ecfan committed:
Minor edits

Commit:
tableau01

MicrosoftGuyJFlo committed:
[Azure AD] AD DS Decom and Redirect content to Windows Server Repo

bsiva committed:
a2a PS

MicrosoftGuyJFlo committed:
[Azure AD] AD DS Decom and Redirect content to Windows Server Repo

Commit:
changed sections to H2, reordered the sections to appear alphabetically, entered the escape character for ClientClaims

GitHubber17 committed:
Merge pull request #38955 from v-stadam/TextAnalyticsLanding

Text analytics landing

lnalepa committed:
Merge pull request #1 from CelesteDG/patch-5

Updated the TOC to include the new file

MicrosoftGuyJFlo committed:
Validation fixes

twooley committed:
[Azure AD B2B] Updated overview article

GitHubber17 committed:
Merge pull request #38497 from ecfan/update-wdl-functions

Refactored Azure Logic Apps WDL functions and main WDL article

rmca14 committed:
Merge pull request #38948 from MashaMSFT/20180424_sqldbaadpics

added some clarity around where to find sql servers

bsiva committed:
Linux kernel support updates

daveba committed:
fix formatting

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into fix-format-overview-breadcrumb

twooley committed:
[Azure AD B2B] Updated overview article

Commit:
minor fixes

rmca14 committed:
Merge pull request #38320 from lnalepa/lesia-build2018

updated v2 registration portal doc

daveba committed:
fix formatting

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into fix-format-overview-breadcrumb

vasivara-MSFT committed:
(ForumsTeam) failure misspelled as faiure - fixes MicrosoftDocs/azure-docs#7725

Commit:
Merge pull request #39093 from srrengar/minorfixes

Minor fixes

PRMerger13 committed:
Merge pull request #38605 from gatneil/vmss-appgw

added App Gateway section to VMSS Networking doc

PRMerger13 committed:
Merge pull request #39095 from vasivara-MSFT/patch-4

(ForumsTeam) failure misspelled fixes MicrosoftDocs/azure-docs#7725

daveba committed:
fix formatting

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into fix-format-overview-breadcrumb

twooley committed:
[Azure AD B2B] Updated overview article

Commit:
arranged the settings alphabetically

JasonWHowell committed:
New landing page

JasonWHowell committed:
New landing page

JasonWHowell committed:
New landing page

cherylmc committed:
format

JasonWHowell committed:
Fixing incoming links that were broken

JasonWHowell committed:
Fixing missing icon

JasonWHowell committed:
Fixing missing icon

JasonWHowell committed:
Update stream-analytics-quick-create-powershell.md

cherylmc committed:
update

JasonWHowell committed:
Update stream-analytics-quick-create-powershell.md

billmath committed:
updating with new download

JasonWHowell committed:
Update stream-analytics-quick-create-powershell.md

JasonWHowell committed:
Fixing missing icon

JasonWHowell committed:
Fixing wrong link

PRMerger17 committed:
Merge pull request #39107 from JasonWHowell/patch-1

Update stream-analytics-quick-create-powershell.md

JasonWHowell committed:
Removing non-MVC tutorials

cherylmc committed:
update

Commit:
removing h2

Commit:
tableau01

daveba committed:
fix formatting

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into fix-format-overview-breadcrumb

mimckitt committed:
Fixes MicrosoftDocs/azure-docs#7648

-Removed "Cloud Service Fundamentals in Microsoft Azure. A comprehensive sample demonstrating a wide range of best practices, produced by the Microsoft Patterns and Practices group" as it pointed to nothing and is not needed anyways 
-Fixed the Azure storage link at the bottom to point correctly to Azure Storage Documentation

PRMerger20 committed:
Merge pull request #39113 from MicahMcKittrick-MSFT/patch-24

Fixes MicrosoftDocs/azure-docs#7648

Commit:
tableau01

daveba committed:
fix formatting

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into fix-format-overview-breadcrumb

Commit:
tableau01

mgblythe committed:
Add gdpr info for logic apps gateway

daveba committed:
fix formatting

daveba committed:
fix formatting

PRMerger18 committed:
Merge pull request #39118 from mgblythe/master

Add gdpr info for logic apps gateway

GitHubber17 committed:
Merge pull request #38385 from cynthn/disk-swap

Swap OS Disk

daveba committed:
fix formatting

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into fix-format-overview-breadcrumb

GitHubber17 committed:
Merge pull request #38974 from v-viinde/netsuiteupdate

USER STORY 1245498 SaaS App Tutorial: Netsuite update

GitHubber17 committed:
Merge pull request #38997 from ash2017/patch-1

Update iot-hub-compare-event-hubs.md

PRMerger10 committed:
Merge pull request #39092 from daveba/fix-format-overview-breadcrumb

[Azure AD MSI] Fix format overview breadcrumb

GitHubber17 committed:
Merge pull request #39058 from MarkusVi/tableau01

tableau01

v-shils committed:
Merge pull request #39059 from JasonWHowell/sa-inputs

Stream Analytics -removing old input information and shuffling content

GitHubber17 committed:
Merge pull request #39067 from v-deken/LandingPage-VideoSearch

Landing page video search

GitHubber17 committed:
Merge pull request #39102 from v-payc/corrections

Corrections

GitHubber17 committed:
Merge pull request #39105 from cherylmc/toc5

format

tfosmark committed:
Merge pull request #37725 from mattbriggs/mdb_20180412_SDK_Ruby

API Profiles Ruby SDK

GitHubber17 committed:
Merge pull request #39082 from MicrosoftGuyJFlo/ADDSRedirectandRemove

[Azure AD] AD DS Decom and Redirect content to Windows Server Repo

Commit:
Merge pull request #37553 from asgang/master

adding France Geos

Commit:
Merge pull request #38395 from gled4er/kaniodimitrov/activity-function-with-multiple-parameters

Passing multiple parameters to Activity Functions

vinynigam committed:
Added URLs

Added URLs

Commit:
Merge pull request #39135 from vinynigam/patch-2

Added URLs

Commit:
Added cross-link.

Commit:
Added cross-link.

Commit:
Edits.

billmath committed:
updating

PRMerger15 committed:
Merge pull request #39140 from jimdial/patch-9

Added cross-link.

PRMerger15 committed:
Merge pull request #39139 from jimdial/patch-8

Added cross-link.

itechedit committed:
Merge pull request #37734 from georgewallace/updatetutorial

automation : updates to update tutorial

rjagiewich committed:
Merge pull request #39081 from bsiva/master

A2A PowerShell support

rwike77 committed:
review feedback

rwike77 committed:
acrolinx fixes

paulshealy1 committed:
Update GPU info for Ubuntu DSVM

Duncanma committed:
Merge pull request #38873 from MicrosoftDocs/TOCTypoFix

Fixing a typo in the TOC

rjagiewich committed:
Merge pull request #39090 from twooley/twb2boverviewquick

[Azure AD B2B] Updated overview article

paulshealy1 committed:
Update DLVM docs with allowed GPU types

GitHubber17 committed:
Merge pull request #38507 from rwike77/cnfeature

cert common name

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

GitHubber17 committed:
Merge pull request #39029 from billmath/compat

staging compat doc

v-alje committed:
Merge pull request #39167 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger18 committed:
Merge pull request #39166 from paulshealy1/patch-3

Update DLVM docs with allowed GPU types

PRMerger18 committed:
Merge pull request #39164 from paulshealy1/patch-2

Update GPU info for Ubuntu DSVM

GitHubber17 committed:
Merge pull request #39103 from JasonWHowell/sa-landingpage

Modernizing the Stream Analytics landing page
